Makefile

```diff
@@ -23,7 +25,7 @@ ifeq ($(DIFF), 1)
 endif
 
 PP_PKG=github.com/mattermost/mattermost-push-proxy/internal/version
-LDFLAGS="-X $(PP_PKG).gitVersion=$(GIT_VERSION) -X $(PP_PKG).gitCommit=$(GIT_HASH) -X $(PP_PKG).gitTreeState=$(GIT_TREESTATE) -X $(PP_PKG).buildDate=$(BUILD_DATE)"
+LDFLAGS="-X $(PP_PKG).gitVersion=$(GIT_VERSION) -X $(PP_PKG).buildHash=$(BUILD_HASH) -X $(PP_PKG).buildTagLatest=$(BUILD_TAG_LATEST) -X $(PP_PKG).buildTagCurrent=$(BUILD_TAG_CURRENT) -X $(PP_PKG).gitTreeState=$(GIT_TREESTATE) -X $(PP_PKG).buildDate=$(BUILD_DATE)"
```

Comment:
Heads up that there are some changes to the Makefile here: https://github.com/mattermost/mattermost-push-proxy/pull/109.

Reply:
Thanks, merged and updated the Makefile.